### DISCUSSION TITLE
Webhook or trigger support when creating new album

### DISCUSSION THREAD

**Discussion by seanpai96** [photoprism/photoprism#2997] [Ideas]
Recently I started to use this as a shared album for my family, which is doing great. And I want to automate the process of informing everyone that "a new album is created, please take a look or upload your own photos". So I think a webhook or trigger script would be great. 

Or can this be implemented by other method?

**Comment by lastzero** [comment]
At this time, you could theoretically listen to the websocket events for this or add database triggers. You could also poll the REST API. But none of that would be as easy to use as webhooks.

**Comment by melyux** [comment]
In the same vein, some ability to run hook scripts when new things are imported from the import folder would be great. No push notification support, but we could roll our own until that comes along by using a hook like that.
